crates/bitwarden-wasm-internal/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "bitwarden-wasm-internal"
+version = "0.1.0"
+publish = false
+
+authors.workspace = true
+edition.workspace = true
+rust-version.workspace = true
+homepage.workspace = true
+repository.workspace = true
+license-file.workspace = true
+keywords.workspace = true
+
+[lib]
+crate-type = ["cdylib"]
+
+[dependencies]
+bitwarden = { workspace = true, features = ["internal", "wasm"] }
+console_error_panic_hook = "0.1.7"
+console_log = { version = "1.0.0", features = ["color"] }
+js-sys = "0.3.68"
+log = "0.4.20"
+serde_json = ">=1.0.96, <2.0"
+wasm-bindgen = { version = "0.2.91", features = ["serde-serialize"] }
```

Comment:
question: should this use the workspace version?

```wasm-bindgen = { workspace = true }```

I haven't seen this syntax before, but I saw that you did this in `crates/bitwarden-core/Cargo.toml`

Reply:
No, we use an explicit version here while the workspace uses ranges.

Reply:
Oh ok, is that because this version is used for building and in that case `wasm-bindgen` and `wasm-bindgen-cli` versions need to be exact, while for the other stuff we can be more relaxed with the version? Do you think that different versions can cause issues? Or should it be fine since we're using the same minor versions?